style.css

```diff
@@ -36,11 +36,70 @@ header {
   padding-left: 25px;
   padding-right: 25px;
   background-color: white;
+  align-items: center;
+}
+
```

Comment:
- [ ]  [OPTIONAL]  I highly recommend fixing this grey space. The body has some margin by default. You can fix this by giving it `margin: 0`. 😄 
![image](https://user-images.githubusercontent.com/92215878/189974420-dc4fbbfb-2a0e-46db-b839-e7ff3e042db7.png)